lib/css/components/_stacks-popovers.less

```diff
@@ -16,27 +16,25 @@
 //  $   BASE STYLES
 //  ----------------------------------------------------------------------------
 .s-popover {
-    visibility: hidden;
-    position: relative;
+    display: none;
+    position: absolute;
     min-width: 12rem;
     max-width: 24rem;
     width: 100%;
     padding: @su12;
-    z-index: @zi-hide;
     opacity: 0;
     border-radius: @br-md;
     border: 1px solid @black-100;
     background-color: @white;
     box-shadow: @bs-sm;
     color: @fc-dark;
     font-size: @fs-body1;
-    transition: opacity 100ms @te-smooth 0s, z-index 0s 100ms, visibility 0s 100ms; // Transition out
 
     &.is-visible {
-        visibility: visible;
+        display: block;
```

Comment:
This change will require some love in production to see if we're breaking things.